config/lib/express.js

```diff
@@ -279,6 +279,7 @@ module.exports.initHelmetHeaders = function (app) {
         '*.twitter.com',
         '*.google-analytics.com',
         '*.gstatic.com', // Google analytics related
+        '*.addthis.com', // addthis.com social media sharing service
```

Comment:
`*.addthis.com` probably has to be added to `childSrc` rules as well.

Check also other CSP warnings from console.log (at least Chrome shows them up).